src/swati/Assignment12/A12_FindFreqChar.java

```diff
@@ -0,0 +1,45 @@
+package swati.Assignment12;
+
+import java.util.Scanner;
+
+public class A12_FindFreqChar {
+
+	int count;
+	int index;
+	void findRep(String name) {
+
+		for (int i = 0; i < name.length(); i++) {
+			int count = 0;
+
+			// System.out.println("----" + ch);
+			for (int j = 0; j < name.length(); j++) {
```

Comment:
here you can aslo use 
indexOf(ch) != lastIndexOf(ch) logic to avoid one more for loop